### DISCUSSION TITLE
Add custom CSS fonts to Vanilla Js app

### DISCUSSION THREAD

**Discussion by abir75mal** [community/community#64883] [Programming Help]
### Body

I'm trying to add a custom font from a .tff or .woff2 files but I don't see the new fonts.
I tried
`
@font-face {
  font-family: "myFont";
  src: url("../assets/myFont.woff2") format("woff2");
  src: url("../assets/myFontTwo.ttf") format("truetype");
}
`

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by artiomtz** [comment]
You have to define each font separately and then use them in an element

body {
  font-family: "myFont", "myFontTwo";
}

@font-face {
  font-family: "myFont";
  src: url("../assets/myFont.woff2") format("woff2");
}

@font-face {
  font-family: "myFontTwo";
  src: url("../assets/myFontTwo.ttf") format("truetype");
}

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
